flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-oceanbase/src/main/java/org/apache/flink/cdc/connectors/oceanbase/sink/OceanBaseDataSinkOptions.java

```diff
@@ -0,0 +1,173 @@
+            ConfigOptions.key("direct-load.port")
+                    .intType()
+                    .defaultValue(2882)
+                    .withDescription("Rpc port number used in direct load.");
+
+    @Experimental
+    public static final ConfigOption<Integer> DIRECT_LOAD_PARALLEL =
+            ConfigOptions.key("direct-load.parallel")
+                    .intType()
+                    .defaultValue(8)
+                    .withDescription("Parallelism of direct load.");
+
+    @Experimental
+    public static final ConfigOption<Long> DIRECT_LOAD_MAX_ERROR_ROWS =
+            ConfigOptions.key("direct-load.max-error-rows")
+                    .longType()
+                    .defaultValue(0L)
+                    .withDescription("Maximum tolerable number of error rows.");
+
+    @Experimental
+    public static final ConfigOption<ObLoadDupActionType> DIRECT_LOAD_DUP_ACTION =
+            ConfigOptions.key("direct-load.dup-action")
+                    .enumType(ObLoadDupActionType.class)
+                    .defaultValue(ObLoadDupActionType.REPLACE)
+                    .withDescription("Action when there is duplicated record in direct load.");
+
+    @Experimental
+    public static final ConfigOption<Duration> DIRECT_LOAD_TIMEOUT =
+            ConfigOptions.key("direct-load.timeout")
+                    .durationType()
+                    .defaultValue(Duration.ofDays(7))
+                    .withDescription("Timeout for direct load task.");
+
+    @Experimental
+    public static final ConfigOption<Duration> DIRECT_LOAD_HEARTBEAT_TIMEOUT =
+            ConfigOptions.key("direct-load.heartbeat-timeout")
+                    .durationType()
+                    .defaultValue(Duration.ofSeconds(30))
+                    .withDescription("Client heartbeat timeout in direct load task.");
+
+    public static final ConfigOption<Boolean> SYNC_WRITE =
+            ConfigOptions.key("sync-write")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Whether to write synchronously.");
+
+    public static final ConfigOption<Duration> BUFFER_FLUSH_INTERVAL =
+            ConfigOptions.key("buffer-flush.interval")
+                    .durationType()
+                    .defaultValue(Duration.ofSeconds(1))
+                    .withDescription(
+                            "The flush interval, over this time, asynchronous threads will flush data. Default value is '1s'. "
+                                    + "If it's set to zero value like '0', scheduled flushing will be disabled.");
+
+    public static final ConfigOption<Integer> BUFFER_SIZE =
+            ConfigOptions.key("buffer-flush.buffer-size")
+                    .intType()
+                    .defaultValue(1000)
+                    .withDescription("Buffer size. Default value is '1000'.");
+
+    public static final ConfigOption<Integer> MAX_RETRIES =
+            ConfigOptions.key("max-retries")
```

Comment:
- All options will be passed to fink-connector-oceanbase and then used. See the following code:
```
        Configuration config = context.getFactoryConfiguration();
        Map<String, String> configMap = config.toMap();
        OptionUtils.printOptions(IDENTIFIER, configMap);
        
        OceanBaseConnectorOptions connectorOptions = new OceanBaseConnectorOptions(configMap);
        String zoneStr = context.getPipelineConfiguration().get(PIPELINE_LOCAL_TIME_ZONE);
        ZoneId zoneId =
                PIPELINE_LOCAL_TIME_ZONE.defaultValue().equals(zoneStr)
                        ? ZoneId.systemDefault()
                        : ZoneId.of(zoneStr);
        return new OceanBaseDataSink(connectorOptions, zoneId);
```
- I removed direct-load related options because the direct-load feature has been removed from flink-sql-connector-oceanbase since version 1.4.